test/integration/crcsuite/crcsuite.go

```diff
@@ -223,15 +224,28 @@ func LoginToOcClusterSucceedsOrFails(expected string) error {
 
 func StartCRCWithDefaultBundleAndDefaultHypervisorSucceedsOrFails(expected string) error {
 
-	cmd := fmt.Sprintf("crc start -b %s -p '%s' --log-level debug", bundleName, pullSecretFile)
+	var cmd string
+
+	if constants.BundleEmbedded() {
+		cmd = fmt.Sprintf("crc start -p '%s' --log-level debug", pullSecretFile)
+	} else {
+		cmd = fmt.Sprintf("crc start -b %s -p '%s' --log-level debug", bundleName, pullSecretFile)
+	}
```

Comment:
Maybe be slightly simpler as:
```
var extraBundleArgs string
if !constants.BundleEmbedded() {
    cmd =  fmt.Sprintf(" -b %s", bundleName
}
cmd =  fmt.Sprintf("crc start -p '%s' --log-level debug %s", pullSecretFile, extraBundleArgs)

Reply:
@cfergeau I changed it as you suggested - not sure why it doesn't say "outdated" above the code snippet...